poi/src/main/java/org/apache/poi/ss/util/CellUtil.java

```diff
@@ -302,40 +398,38 @@ public static void copyCell(Cell srcCell, Cell destCell, CellCopyPolicy policy,
 
     /**
      * Take a cell, and align it.
-     *
+     * <p>
      * This is superior to cell.getCellStyle().setAlignment(align) because
      * this method will not modify the CellStyle object that may be referenced
      * by multiple cells. Instead, this method will search for existing CellStyles
      * that match the desired CellStyle, creating a new CellStyle with the desired
      * style if no match exists.
      *
-     * @param cell the cell to set the alignment for
+     * @param cell  the cell to set the alignment for
      * @param align the horizontal alignment to use.
-     *
      * @see HorizontalAlignment for alignment options
      * @since POI 3.15 beta 3
      */
     public static void setAlignment(Cell cell, HorizontalAlignment align) {
-        setCellStyleProperty(cell, ALIGNMENT, align);
+        setCellStyleProperty(cell, CellPropertyType.ALIGNMENT, align);
     }
 
     /**
      * Take a cell, and vertically align it.
-     *
+     * <p>
      * This is superior to cell.getCellStyle().setVerticalAlignment(align) because
      * this method will not modify the CellStyle object that may be referenced
      * by multiple cells. Instead, this method will search for existing CellStyles
      * that match the desired CellStyle, creating a new CellStyle with the desired
      * style if no match exists.
      *
-     * @param cell the cell to set the alignment for
+     * @param cell  the cell to set the alignment for
```

Comment:
Can you remove the whitespace you are adding to all the javadoc? Please do not modify the existing style of the code.